src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,147 @@
+| [intl](https://pub.dartlang.org/packages/intl) | Internationalization and localization facilities, and support for plurals and genders, date and number formatting and parsing, and bidirectional text. | Bidi, DateFormat, MicroMoney, TextDirection |
+| [logging](https://pub.dartlang.org/packages/logging) | A configurable mechanism for adding message logging to your application. | LoggerHandler, Level, LogRecord |
+| [mockito](https://pub.dartlang.org/packages/mockito) | A popular framework for mocking objects in tests. Especially useful if you are writing tests for dependency injection. Used in conjunction with the [test](https://pub.dartlang.org/packages/test) package. | Answering, Expectation, Verification |
+| [path](https://pub.dartlang.org/packages/path) | Common operations for manipulating different types of paths. For more information, [Unboxing Packages: path](http://news.dartlang.org/2016/06/unboxing-packages-path.html). | absolute(), basename(), extension(), join(), normalize(), relative(), split() |
+| [quiver](https://pub.dartlang.org/packages/quiver) | Utilities that make using core Dart libraries more convenient. Some of the libraries where Quiver provides additional support include async, cache, collection, core, iterables, patterns, and testing. | CountdownTimer, (quiver.async), MapCache (quiver.cache), MultiMap, TreeSet (quiver.collection), EnumerateIterable (quiver.iterables), center(), compareIgnoreCase(), isWhiteSpace() (quiver.strings)  |
+| [shelf](https://pub.dartlang.org/packages/shelf) | Web server middleware for Dart. Shelf makes it easy to create and compose web servers, and parts of web servers. | Cascade, Pipeline, Request, Response, Server |
+| [stack_trace](https://pub.dartlang.org/packages/stack_trace) | Methods for parsing, inspecting, and manipulating stack traces produced by the underlying Dart implementation. Also provides functions to produce string representations of stack traces in a more readable format than the native StackTrace implementation. For more information, [Unboxing Packages: stack_trace](http://news.dartlang.org/2016/01/unboxing-packages-stacktrace.html). | Trace.current(), Trace.format(), Trace.from() |
+| [stagehand](https://pub.dartlang.org/packages/stagehand) | A Dart project generator. WebStorm and IntelliJ use Stagehand templates when you create a new application, but you can also use the templates from the command line via [pub global activate](https://www.dartlang.org/tools/pub/cmd/pub-global). | Templates include console-full, console-simple, server-shelf, web-angular, and web-simple. |
+| [test](https://pub.dartlang.org/packages/test) | A standard way of writing and running tests in Dart. | expect(), group(), test() |
+{:.table .table-striped .nowrap}
+
+To find more packages, see [pub.dartlang.org](https://pub.dartlang.org/).
+
+## Packages that correspond to SDK libraries
+
+These "expansion pack" libraries have the same name as their counterpart in the SDK.
```

Comment:
How about dropping the first sentence and mentioning/linking to the corresponding library in each description? So:

Each of these "expansion pack" libraries....

And then, in the table, something like:

Expands on dart:async, adding utilities for working with asynchronous computations. For more information, see Unboxing Packages: async part 1, part 2, and part 3.